config.yaml

```diff
@@ -22,7 +22,7 @@ external_services:
   # Uncomment istio_identity_domain to set a different value. This value must match the Istio configuration.
   # Default is "svc.cluster.local" (which matches Istio's default)
   # istio_identity_domain: svc.cluster.local
-  jaeger:
+  tracing:
```

Comment:
This entire config.yaml should be deleted. This whole `external_services` section is obsolete and I'm pretty sure won't even work.

Reply:
I found config.yaml referrenced in 2 files.
https://github.com/kiali/kiali/blob/8df7c28c68b367b3e75b47b12921556702b818ab/business/iter8.go#L421
https://github.com/kiali/kiali/blob/c49238fc81b486f199c06e7ce69efed5081a3069/make/Makefile.build.mk#L89

Should I only fix external_services section?

Reply:
GetAnalyticPort(iter8.go) is not used in this repository, so perhaps it can be removed.
About Makefile, if not configuration file specified, kiali cli will use environment variables and work, so it can be removed.

Reply:
kiali/business/iter8.go isn't referencing that file - it is referencing the file mounted from the ConfigMap - that is unrelated.
The Makefile "run" target is interesting... maybe someone is using that somewhere, but I don't know who or why.

Reply:
> kiali/business/iter8.go isn't referencing that file - it is referencing the file mounted from the ConfigMap - that is unrelated.

Ok, thank you.

> The Makefile "run" target is interesting... maybe someone is using that somewhere, but I don't know who or why.

The README.adoc introduces it in Running Standalone section.
https://github.com/kiali/kiali/blob/master/README.adoc#running-standalone